tests/files/tests.py

```diff
@@ -120,18 +120,36 @@ def test_file_iteration_with_text(self):
         f = File(StringIO('one\ntwo\nthree'))
         self.assertEqual(list(f), ['one\n', 'two\n', 'three'])
 
+    def test_readable(self):
+        with tempfile.TemporaryFile() as temp, File(temp, name='something.txt') as test_file:
+            self.assertTrue(test_file.readable())
+        self.assertFalse(test_file.readable())
+
+    def test_writable(self):
+        with tempfile.TemporaryFile() as temp, File(temp, name='something.txt') as test_file:
+            self.assertTrue(test_file.writable())
+        self.assertFalse(test_file.writable())
+        with tempfile.TemporaryFile('rb') as temp, File(temp, name='something.txt') as test_file:
+            self.assertFalse(test_file.writable())
+
     def test_seekable(self):
-        """
-        File.seekable() should be available on Python 3.
-        """
-        with tempfile.TemporaryFile() as temp:
-            temp.write(b"contents\n")
-            test_file = File(temp, name="something.txt")
-            if six.PY2:
-                self.assertFalse(hasattr(test_file, 'seekable'))
-            if six.PY3:
-                self.assertTrue(hasattr(test_file, 'seekable'))
-                self.assertTrue(test_file.seekable())
+        with tempfile.TemporaryFile() as temp, File(temp, name='something.txt') as test_file:
+            self.assertTrue(test_file.seekable())
+        self.assertFalse(test_file.seekable())
+
+    def test_io_wrapper(self):
+        content = "vive l'été\n"
+        with tempfile.TemporaryFile() as temp, File(temp, name='something.txt') as test_file:
+            test_file.write(content.encode('utf-8'))
+            test_file.seek(0)
+            wrapper = TextIOWrapper(test_file, 'utf-8', newline='\n')
+            self.assertEqual(wrapper.read(), content)
+            wrapper.write(content)
+            wrapper.seek(0)
```

Comment:
Windows Python 2.7 crash with `IOError: [Errno 0] Error` here and I've no clue what could be wrong...